### ISSUE TITLE
timestamp_ms() truncates on python 2

### ISSUE THREAD

**Issue by tommyolofsson** [kisom/pypcapfile#25]
Different resolution on python 2 and 3 since 0.12.0. Not sure if this is intentional.

**Comment by asergi** [comment]
@kisom 
```
>>> pkt.timestamp_us
232646
>>> pkt.timestamp_ms
232
```
or 
```
>>> pkt.timestamp_ms
232.646
```

**Comment by Don42** [comment]
That is not intentional. This bug was introduced when I fixed #12.
I'm going to create a PR to fix this. I'm only running code on py3 and forgot the difference.
